tests/components/test_logbook.py

```diff
@@ -271,13 +271,51 @@ def test_exclude_automation_events(self):
         entity_id2 = 'automation.my_automation_rule_2'
         entity_id2 = 'sensor.blu'
 
-        eventA = ha.Event(logbook.EVENT_LOGBOOK_ENTRY, {
+        eventA = ha.Event(logbook.EVENT_AUTOMATION_TRIGGERED, {
             logbook.ATTR_NAME: name,
             logbook.ATTR_MESSAGE: message,
             logbook.ATTR_DOMAIN: domain,
```

Comment:
Message and Domain aren't part of the new event.  You can see it defined [here](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/automation/__init__.py#L290-L293)

Reply:
Many thanks! Updated.